src/scoreSheetBot.py

```diff
@@ -1246,6 +1246,9 @@ async def flair(self, ctx: Context, member: discord.Member, *, new_crew: str = N
         if flairing_crew.overflow:
             overflow_server = discord.utils.get(self.bot.guilds, name=OVERFLOW_SERVER)
             of_after = set(overflow_server.get_member(member.id).roles)
+        flaired_crew = crew_members(flairing_crew, self)
+        if len(flaired_crew) == 40:
```

Comment:
Nit, just use `len(crew_members(flairing_crew, self))` instead of making a variable and using it once.